diff --git a/CMakeLists.txt b/CMakeLists.txt
index 523953f9..2dd761e9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -105,13 +105,11 @@ if (NOT BUILD_SKYRIMVR)
 		"$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
 )
 else()
-	set(OPENVR_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/extern/openvr/headers)
 	target_include_directories(
 	"${PROJECT_NAME}"
 	PUBLIC
 		"$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
 		"$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
-		"$<BUILD_INTERFACE:${OPENVR_INCLUDE_DIR}>"
 	)
 endif()
 
@@ -167,3 +165,8 @@ install(
 		"include/SKSE"
 	DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
 )
+install(
+	FILES
+		"include/csv.h"
+	DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
+)
